pkgs/tools/misc/fontforge/default.nix

```diff
@@ -67,16 +116,20 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall =
-    # get rid of the runtime dependency on python
-    lib.optionalString (!withPython) ''
-      rm -r "$out/share/fontforge/python"
-    '';
+    ''
+      wrapProgram $out/bin/fontforge --suffix PYTHONPATH : "${py}/${py.sitePackages}"
+    ''
```

Comment:
And this should be in a withPython conditional too

Reply:
I would like to first know what `$out/share/fontforge/python` does. Do you know that?

Reply:
No, I haven't jumped into the fontforge deep end yet. 🙂

Reply:
OK. By the way I shoudn't have replied here. Let's discuss this outside, directly under the PR.